### ISSUE TITLE
Trace Log Listener

### ISSUE THREAD

**Issue by LSTANCZYK** [serilog/serilog#457]
How would I go about creating a trace listener to work with serilog so I can include traces produced by third party libraries into the log? 

In my application I use serilog, but I am also using an application framework that writes logs to trace. My goal is to have them in one place, So i have logger configured to write to the database and would like to also set up a trace listener to pipe trace info to serilog logger (hope this explanation makes sense).

**Comment by IanYates** [comment]
Hey, I knocked one up a little while ago whilst trying to debug some SignalR stuff.  I'll try to dig it out later today and put it in a Gist.  I guess it would be a good thing for a separate project eventually...

**Comment by LSTANCZYK** [comment]
That would be great. I am over budget on the project that needs it so would be great to just get one you baked :) Any idea when you'd be able to dig it up?

**Comment by IanYates** [comment]
My first Gist :open_hands:   ...  https://gist.github.com/IanYates/7f31abe19610ca653a3b

Note that it's actually an adapter to Castle Windsor's ILogger (which I haven't yet removed from my project).  It also uses a couple of little extension methods like `.UpliftTo` which creates a logger that uplifts events of low importance up to a specified importance.  But it's pretty complete.

I borrowed a chunk of the code from some StackOverflow posts and a few Gists I looked at.  I normally keep URLs in the code but didn't seem to this time.
